integration_tests/test_python_plugin.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2015 Canonical Ltd
+# Copyright (C) 2015, 2016, 2017 Canonical Ltd
```

Comment:
well since I started a review, please make this
`# Copyright (C) 2015-2017 Canonical Ltd`